extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -133,17 +141,29 @@ class ConsumeWindowsEventLog : public core::Processor {
   std::wstring wstrQuery_;
   std::string regex_;
   bool resolve_as_attributes_;
```

Comment:
I suppose this should be initialized as well.

Reply:
initialized